assets/mods/simplify/postloadModule.js

```diff
@@ -42,20 +42,36 @@ import * as patchSteps from './lib/patch-steps-es6.js';
 		/**
 		 * Generates patches for a pair of given objects or files.
 		 * @param {object} target
-		 * @param {object|array} patch
-		 * @param {string} modbase
+		 * @param {object|array} patchData
+		 * @param {{mod: Mod, path: string}} patch
 		 * @return {Promise<any>} result
 		 */
-		async _applyPatch(target, patch, modbase) {
-			await patchSteps.patch(target, patch, async (fromGame, url) => {
-				if (fromGame) {
-					// Import (game file)
-					return await this.loadJSONPatched(igroot + url);
-				} else {
-					// Include (mod file)
-					return await this.loadJSON(modbase + url);
+		async _applyPatch(target, patchData, patch) {
+			const debugState = new CustomDebugState();
+			debugState.setPatch(patch);
+			debugState.addFile([true, patch.path]);
+			await patchSteps.patch(target, patchData, async (fromGame, url) => {
+				let data = null;
```

Comment:
Why was the content of the callback changed? It is less readable, less performant and not really future proof either.

Reply:
"fromGame" was future-proofed on the library side by using true, false and string as values (as sources the application couldn't understand would be interpreted the wrong way anyway)
This presumably then converts true & false to string and bases stuff of of that, but it's not actually doing it properly since the format emitted by the library has no redundant ":"